src/main/java/org/traccar/protocol/TranSyncProtocol.java

```diff
@@ -0,0 +1,22 @@
+package org.traccar.protocol;
```

Comment:
Please make sure to add OSS license notices to all non test files. You can use your name or mine if you don't want to share your details.